lib/circuitry/config/shared_settings.rb

```diff
@@ -0,0 +1,39 @@
+require 'logger'
+
+module Circuitry
+  class ConfigError < StandardError; end
+
+  module Config
+    module SharedSettings
```

Comment:
I think this module should `include Virtus::Model` directly since none of the `attribute` calls will work without it.